subjects/atm-management-system/audit/README.md

```diff
@@ -6,73 +6,73 @@
 
 ##### Open the application and re-register the user `"Alice"`.
 
-###### Did it display an error message stating that this user already exist?
+###### Did the application display an error message stating that this user already exists?
 
 ##### Open the file `"./data/users.txt"`.
 
-###### Are all the user's names unique? (ex: no repetition on the name Alice)
+###### Are all the user names unique? (ex: no repetition on the name Alice)
 
 ##### Try and login as `"Alice"`.
 
 ###### Was Alice able to enter the main menu?
 
-##### Try to create two accounts using the user Alice, then select the option `"Update information of account"` and select an account number that does not exist for Alice.
+##### Try to create two accounts with the user Alice, then select the option `"Update account information"` and select an account number that does not exist for Alice.
```

Comment:
This is fine!
just want to let you know that i already changed it in the project that is given to the students as well.